app/models/pet.rb

```diff
@@ -1,4 +1,31 @@
 class Pet < ApplicationRecord
+  # == Schema Information
+  #
+  # Table name: dogs
```

Comment:
Oddly for some reason its still picking up the old annotation with dogs. Should be pets

Reply:
I believe this is caused by the state of the database still holding dogs as a table. I think adjusting our migrations should fix this.